src/core/aws-server.ads

```diff
@@ -334,6 +336,9 @@ private
 
    type Slot is record
       Sock                  : Socket_Access;
+      Wait_For_Data         : aliased Net.Poll_Events.Set (2);
+      Wait_Breaker          : Net.Std.Socket_Type;
```

Comment:
Breaker?

Reply:
Yes, to break the waiting call.

Reply:
A signaling socket? At least please put a little comment here as it was not trivial to understand when I have reviewed the changes.

Reply:
Added comment at the end of record declaration. Removed obsolete comments there by the way.